main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	entities "main/entities"
+	handlers "main/handlers"
+	utils "main/utils"
+	"net/http"
+)
+
+func main() {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/books", handlers.HandleBookRequests)
+
+	go func() {
+		runTest()
+	}()
+
+	http.ListenAndServe(":8080", mux)
+}
+
+func runTest() {
+	fmt.Println("=================================")
+	fmt.Println("OUR JSON ON START")
+	fmt.Println("=================================")
+	getBooksData()
+
+	fmt.Println("=================================")
+	fmt.Println("ADDING NEW BOOK")
+	fmt.Println("=================================")
+	newBook := entities.Book{
+		Name:   "Refactoring",
+		Author: "Martin Fowler",
+		Year:   1999,
+	}
+	postBooksData(newBook)
+
+	fmt.Println(" ")
+	fmt.Println("=================================")
+	fmt.Println("OUR JSON AFTER ADDING NEW BOOK")
+	fmt.Println("=================================")
+	getBooksData()
+}
+
+func getBooksData() {
+	requestURL := "http://localhost:8080/books"
+	client := &http.Client{}
+
+	r, _ := http.NewRequest("GET", requestURL, nil)
```

Comment:
```suggestion
	r, _ := http.NewRequest(http.MethodGet, requestURL, nil)
```
and in all other cases when you use string do the same pls

Reply:
there are all method constants in 'http' package